include/autocluster.hrl

```diff
@@ -56,6 +56,8 @@
          {config, k8s_namespace_path,    "K8S_NAMESPACE_PATH",     "/var/run/secrets/kubernetes.io/serviceaccount/namespace",
                                                                                  string,  false},
          {config, k8s_service_name,      "K8S_SERVICE_NAME",       "rabbitmq",   string,  false},
+         {config, k8s_address_type,      "K8S_ADDRESS_TYPE",       "ip",         string,  false},
```

Comment:
If the options here are `ip` and `hostname`, please leave a comment mentioning that.